### DISCUSSION TITLE
Resend after SPAM is change

### DISCUSSION THREAD

**Discussion by Igcorreia** [postalserver/postal#1448] [Help with using Postal]
Hi, is there a way to resend emails after the spam threshold is updated?

I GET the following message:
The message is likely spam. The threshold is 5.0 and the message scored 1.2.

![image](https://user-images.githubusercontent.com/1076409/122913973-0cc5ca00-d352-11eb-962b-dadd4501e007.png)

**Comment by josealgardataborges** [comment]
@willpower232 please advise... Is this possible?

**Comment by willpower232** [comment]
If an email has hard failed, it will not be tried again. You can click on the redeliver message button in your screenshot.
